api/src/main/java/org/pmiops/workbench/survey/NewUserSatisfactionSurveyServiceImpl.java

```diff
@@ -135,15 +141,23 @@ public void emailNewUserSatisfactionSurveyLinks() {
               "%s?surveyCode=%s",
               workbenchConfigProvider.get().server.uiBaseUrl,
               dbNewUserSatisfactionSurveyOneTimeCode.getId().toString());
+
       try {
         mailService.sendNewUserSatisfactionSurveyEmail(user, surveyLink);
       } catch (MessagingException e) {
-        throw new ServerErrorException(
+        errorCount++;
+        logger.log(
```

Comment:
we already log if the email fails in MailService.sendWithRetries, I don't think we need to log again here for each failure.

Reply:
makes sense